types/ws/ws-tests.ts

```diff
@@ -19,7 +19,8 @@ import * as https from 'https';
 
 {
     const wss = new WebSocket.Server({port: 8081});
-    wss.on('connection', (ws, req) => {
+    wss.on('connection', function(ws, req) {
+        console.log('port: %s', this.options.port);
```

Comment:
Why you have add console log ?

Reply:
I wanted to test the type of `this` by trying to access its `Websocket.Server.options` property. I assumed it's okay to use a `console.log` since there were already some `console.log`s in the code. Should I use something else?